assignments/array-methods.js

```diff
@@ -55,29 +55,64 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
+
```

Comment:
Great job on the array methods. They ran without errors and I appreciate the comments. :-)